sass/_stylesheet.sass

```diff
@@ -0,0 +1,291 @@
+// Slate Theme for GitHub Pages
+// by Jason Costello, @jsncostello
+
+// MeyerWeb Reset
+html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video
+    margin: 0
+    padding: 0
+    border: 0
+    font: inherit
+    vertical-align: baseline
+
+// HTML5 display-role reset for older browsers
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section
+    display: block
+
+ol, ul
+    list-style: none
+
+table
+    border-collapse: collapse
+    border-spacing: 0
+
+// Theme Styles
+body
+    box-sizing: border-box
+    color: #373737
+    background: #212121
+    font-size: 16px
+    font-family: 'Lato', sans-serif
+    line-height: 1.5
+
+h1, h2, h3, h4, h5, h6
+    margin: 10px 0
+    font-weight: 700
+    color: #222222
+    letter-spacing: -1px
+
+h1
+    font-size: 36px
+    font-weight: 700
+
+h2
+    padding-bottom: 10px
+    font-size: 32px
+    background: url('../images/bg-hr.png') repeat-x bottom
+
+h3
+    font-size: 24px
+
+h4
+    font-size: 21px
+
+h5
+    font-size: 18px
+
+h6
+    font-size: 16px
+
+p
+    margin: 10px 0 15px 0
```

Comment:
oh, no, sorry should have said this one is a basic github stylesheet that I converted to sass, I didn't actually write this